modules/guides/guides.info.yml

```diff
@@ -4,3 +4,4 @@ package: Custom
 
 type: module
 core: 8.x
+version: 8.x-1.0
```

Comment:
Don't add such lines, pls. It's the job of the script that's gonna create the release.